apps/src/applab/redux/applab.js

```diff
@@ -44,10 +72,34 @@ function interfaceMode(state, action) {
   }
 }
 
+function redirectDisplay(state, action) {
+  state = state || [];
```

Comment:
This might just be my unfamiliarity with js, but this seems like js magic. Might be worthwhile to expand this out so it's more clear what's going on here.

Reply:
"this" being 
state = state || []

Reply:
This is a pretty common way we initialize state in reducers, so I'd be nervous to deviate from this practice in just this reducer. But the fact that it's not clear might be a good jumping off point for further discussion on whether we should refactor this initialization procedure across our whole codebase.